plasmapy/utils/tests/test_checks.py

```diff
@@ -252,6 +255,12 @@ def test__check_relativisitc_valid(speed, betafrac):
 def test__check_relativistic_errors(speed, betafrac, error):
     with pytest.raises(error):
         _check_relativistic(speed, 'f', betafrac=betafrac)
+        
+        
+@pytest.mark.parametrize("speed, betafrac, warning", relativistic_warning_examples)
+def test__check_relativistic_warnings(speed, betafrac, warning):
+    with pytest.warns(warning):
+        _check_relativistic(speed, 'f', betafrac=betafrac)
```

Comment:
What was your reasoning for passing the type of warning as an argument? Is this just for testing or is it part of the custom warning structure?

Reply:
Oh, this is a good thing to bring up.  We've been having the warning or exception be an argument for cases where we are testing multiple different warnings or exceptions in one test.  Since this is only testing one warning, we could probably change `with pytest.warns(warning)` to `with pytest.warns(RelativityWarning)` and then not need to have `warning` as an argument to `test__check_relativistic_warnings`.  

We've refactored the tests quite a bit lately, so I'm wondering if we've been having the warning or exception as an argument when only one warning or exception is being tested.  I do really like `pytest.mark.parametrize` even though most of the time I misspell it as `parameterize`!